components/search/MoreFiltersDropdown.vue

```diff
@@ -18,70 +18,85 @@
         {{ selectedOptionsCount }}
       </span>
     </template>
-    <b-dropdown-group class="more-facets-wrapper">
-      <b-dropdown-form v-if="showDateFilter">
+    <b-dropdown-form>
```

Comment:
The entire more filters dropdown is one form and should be marked up as such. This has the additional benefit of permitting use of the `Enter` key to submit the form.